.github/scripts/release_images.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Initialize array to store digest SHAs
+declare -a digests
+
+# Fetch all docker image names
+image_names=$(gh api \
+  -H "Accept: application/vnd.github+json" \
+  -H "-GitHub-Api-Version: 2022-11-28" \
+  -H "Authorization: Token ${GITHUB_TOKEN}" \
+  --paginate "/orgs/boozallen/packages?package_type=container" | jq -r '.[] | select((.name | startswith("aissemble")) and (.name | endswith("-chart") | not)) | .name')
+
+# For each docker image, find all release versions, excluding 1.7.0(-arm64/amd64)
+for name in $image_names; do
+  release_versions=$(gh api \
+    -H "Accept: application/vnd.github+json" \
+    -H "-GitHub-Api-Version: 2022-11-28" \
+    -H "Authorization: Token ${GITHUB_TOKEN}" \
+    --paginate "/orgs/boozallen/packages/container/${name}/versions" \
+    | jq -r '.[] | .metadata.container.tags[] | select(test("^\\d+\\.\\d+\\.\\d?$"))' \
+    | jq -R -s 'split("\n") | map(select(length > 0)) | map(select(. != "1.7.0" and . != "1.7.0-arm64" and . != "1.7.0-amd64"))')
+
+  # Loop through all release versions
+  for version in $(echo "$release_versions" | jq -r '.[]'); do
+    echo "Processing release image ${name}:${version}"
+
+    # Fetch the base manifest SHA
+    # Inspect command will output Name, MediaType, Digest in the first three lines
+    # so we can use a regex to pull out the SHA
+    manifest_base_sha=$(docker buildx imagetools inspect "ghcr.io/boozallen/${name}:${version}" | head -n 3 | sed -n 's/^Digest: *//p')
```

Comment:
I: ideally we would use something more straightforward like `docker buildx imagetools inspect ghcr.io/boozallen/${name}:${version} --format "{{json .Manifest}}" | jq -r .digest` to pull the manifest base SHA. However, because we have buildx's provenance parameter set to `min` (aka true, the default value), there are two extra manifests that are added to the list with `os` and `architecture` set to `unknown`. 
This causes the command to output an error like the following:
```
ERROR: failed to copy: httpReadSeeker: failed open: content at https://ghcr.io/v2/boozallen/aissemble-spark/manifests/sha256:c0ea773c38265bf3a80a133211aba2468dec87b8ce5b341d82611e2c81252147 not found: not found
```
where the referenced SHA points to the attestation/provenance.

The workaround here is to search the first 3 lines of the `inspect` output for the line beginning with `Digest:` and set that to the `manifest_base_sha`. An example of the first 3 lines below:
``` 
Name:      ghcr.io/boozallen/aissemble-spark:1.8.0
MediaType: application/vnd.oci.image.index.v1+json
Digest:    sha256:fb575ed4468a7828a23a8a7d2608977acc59a67e0dd36f9f2608316301554a95
```

Another potential workaround would be to use the GitHub API:
```
gh api \
        -H "Accept: application/vnd.github+json" \
        -H "-GitHub-Api-Version: 2022-11-28" \
        -H "Authorization: Token ${GITHUB_TOKEN}" \
        --paginate  /orgs/boozallen/packages/container/${name}/versions | jq '.[] | select(.metadata.container.tags[] == ${version}) | .name'
```
However, this would require `(# images * # release versions)` total API calls

Reply:
Do we have a Github issue we can link for this bug in the `imagetools inspect` tool? Maybe [buildx#1509](https://github.com/docker/buildx/issues/1509)? Though it looks like that issue has been closed.  I wonder if they have an open issue that would address the JSON output.